repo/packages/A/alluxio-enterprise/0/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "packagingVersion": "3.0",
+  "name": "alluxio-enterprise",
+  "version": "1.4.0",
+  "maintainer": "info@alluxio.com",
+  "description": "Alluxio Enterprise Edition (AEE). Please obtain a license by visiting: https://alluxio.com/products \n\n Alluxio, formerly Tachyon, is the world's first system that unifies disparate storage systems at memory speed. With Alluxio, enterprises can manage data efficiently, accelerate business analytics, and ease the adoption of hybrid cloud. \n\n Please refer to https://alluxio.com/docs/ for a detailed operation guide.",
```

Comment:
@ryadav88 the install guide will be available at alluxio.com/docs (already linked) once i push that live